Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -314,6 +314,8 @@ void TextEditor::mousemove_event(MouseEvent& event)
 
     if (m_ruler_visible && (ruler_rect_in_inner_coordinates().contains(event.position()))) {
         set_override_cursor(Gfx::StandardCursor::None);
+    } else if (m_gutter_visible && (gutter_rect_in_inner_coordinates().contains(event.position()))) {
```

Comment:
These (sure are (a lot (of (parentheses)))) :^) can we drop one set for the right hand side? Might want to do that for the one above as well.

Reply:
You're right, I just copy-pasted without really looking too hard. Removed both of them, thanks!